loopdb/sqlite.go

```diff
@@ -335,9 +336,13 @@ func parseSqliteTimeStamp(dateTimeStr string) (time.Time, error) {
 			dateTimeStr)
 	}
 
-	datePart, timePart := parts[0], parts[1]
+	dateTimeStr, err := replaceFaultyYear(parts[0], dateTimeStr)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("%v: unable to replace faulty "+
+			"year: %v", dateTimeStr, err)
+	}
 
-	return parseTimeParts(datePart, timePart)
+	return time.Parse("2006-01-02 15:04:05.99999 -0700 MST", dateTimeStr)
```

Comment:
Instead of this, I'm wondering if we should try to instead parse the time based on the supported RFC's as documented here: https://pkg.go.dev/time#pkg-constants